lang/it.json

```diff
@@ -0,0 +1,12 @@
+{
+  "No video has been loaded": "Nessun video caricato",
+  "Could not download the video": "Impossibile eseguire il download dei video",
+  "The video connection was lost, please confirm you are connected to the internet": "La connessione al video è andata persa. Verificare di essere connessi a Internet",
+  "The video is bad or in a format that cannot be played on your browser": "Il video è corrotto oppure è in un formato non riproducibile dal browser in uso",
+  "This video is either unavailable or not supported in this browser": "Questo video non è disponibile oppure non è supportato dal browser in uso",
+  "Error Code": "Codice di Errore",
+  "Technical details": "Dettagli tecnici",
+  "The video download was cancelled": "Nessun video caricato",
+  "The video you are trying to watch is encrypted and we do not know how to decrypt it": "Il video che si sta cercando di riprodurre è codificato in un formato non riconosciuto",
+  "An unanticipated problem was encountered, check back soon and try again": "Si è verificato un problema imprevisto. Riprovare più tardi"
+}
```

Comment:
The last three fields are in plugin.js but not in the other lang files.  I had them translated so figured it was better to include them.